mono/mini/aot-compiler.c

```diff
@@ -4427,7 +4427,13 @@ method_has_type_vars (MonoMethod *method)
 static
 gboolean mono_aot_mode_is_full (MonoAotOptions *opts)
 {
-	return opts->mode == MONO_AOT_MODE_FULL;
+	return opts->mode == MONO_AOT_MODE_FULL || opts->mode == MONO_AOT_MODE_INTERP;
```

Comment:
This implies that if you do `--aot=interp` then all the methods are going to be AOTted, even generics with gshared, gsharedvt, etc. I don't think that's the semantic we want from that. From my understanding interpAOT is supposed to be trampolines + wrappers that can't be interpreted.